Added initial draft charter. (#2)
jupyter/docs: charter.md

> The Docs Working Group should exist as a support/helper and advisory group to all the subprojects in Jupyter. Here are some additional details to clarify the WG's scope and role:

Reviewer: ```suggestion
# Docs Working Group Charter (Proposed)

The Docs Working Group serves to support, assist, and advise Jupyter subprojects on documentation best practices.
```

> 

Reviewer: ```suggestion
## Goals

The Docs Working Group's goals should serve the end users of Jupyter projects.
```

Reviewer: Move the later goals/mission up to this section.

> ### The WG's scope and role:

Reviewer: ```suggestion
## Docs WG's scope and role
```

> - Help write docs (inside the bounds of each subproject's governance)

Reviewer: ```suggestion
- Create a markdown file that links to all projects and their user-facing documentation (This markdown file will help provide an easy way to link out to the different subproject's docs).
- Help write docs (inside the bounds of each subproject's governance)
```
